src/SFA.DAS.ApplyService.Data/AssessorRepository.cs

```diff
@@ -226,7 +226,7 @@ public async Task<List<ModerationApplicationSummary>> GetApplicationsInModeratio
                             , ModerationStatus
 	                        FROM Apply apply
 	                        INNER JOIN Organisations org ON org.Id = apply.OrganisationId
-	                        WHERE {InModerationApplicationsWhereClause}
```

Comment:
Can this be changed back to using this where clause (update it if appropriate) as it's also required for the count for the dashboards.

Reply:
ok, I thought I'd merged right